sourced/ml/utils/engine.py

```diff
@@ -7,26 +7,57 @@
 from sourced.engine import Engine  # nopep8
 
 
-def create_spark(session_name, kwargs):
+class SparkDefault:
+    """
+    Default arguments for create_spark function and __main__
+    """
+    MASTER_ADDRESS = "local[*]"
+    LOCAL_DIR = "/tmp/spark"
+    CONFIG = []
+    PACKAGE = []
+
+
+def create_spark(session_name,
+                 spark=SparkDefault.MASTER_ADDRESS,
+                 spark_local_dir=SparkDefault.LOCAL_DIR,
+                 config=SparkDefault.CONFIG,
+                 package=SparkDefault.PACKAGE,
+                 **kwargs):
     log = logging.getLogger("spark")
-    log.info("Starting %s on %s", session_name, kwargs.spark)
-    builder = SparkSession.builder.master(kwargs.spark).appName(session_name)
+    log.info("Starting %s on %s", session_name, spark)
+    builder = SparkSession.builder.master(spark).appName(session_name)
     builder = builder.config(
-        "spark.jars.packages", ",".join(kwargs.package))
-    builder = builder.config("spark.local.dir", kwargs.spark_local_dir)
-    for cfg in kwargs.config:
+        "spark.jars.packages", ",".join(package))
+    builder = builder.config("spark.local.dir", spark_local_dir)
+    for cfg in config:
         builder = builder.config(*cfg.split("=", 1))
     session = builder.getOrCreate()
     session.sparkContext.setLogLevel(logging._levelToName[log.getEffectiveLevel()])
```

Comment:
I had to change it yesterday in `develop`, please update. Sorry, really had to.

Just copy-paste, no need to rebase.

Reply:
copy-paste yours or my changes?

Reply:
Of course mine, yours are here in the PR :-D